spec.html

```diff
@@ -32103,6 +32115,55 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          ECMAScript identifies time zones using the Zone and Link names of the IANA Time Zone Database <a href="https://www.iana.org/time-zones/">https://www.iana.org/time-zones/</a>.
+          These names are called <dfn variants="time zone identifier">time zone identifiers</dfn>, and are Strings composed entirely of characters from the Basic Latin Unicode block.
+          Time zone identifiers are case-insensitive and so are compared using ASCII case-insensitive comparisons, but time zone identifiers returned by ECMAScript built-in objects always use the casing found in the IANA Time Zone Database.
+          A <dfn id="primary-time-zone-identifier" variants="primary time zone identifiers">primary time zone identifier</dfn> is a Zone name, and a <dfn variants="non-primary time zone identifiers">non-primary time zone identifier</dfn> is a Link name, respectively, in the IANA Time Zone Database except as specifically overridden by AvailableTimeZoneIdentifiers.
+          The set of Zone and Link identifiers that are supported by an ECMAScript implementation are called available time zone identifiers.
+        </p>
+
+        <p>In the IANA Time Zone Database, the UTC time zone is represented by the Zone *"Etc/UTC"*, but for backwards compatibility and brevity reasons, *"UTC"* is the primary identifier used for this time zone in ECMAScript, while *"Etc/UTC"*, *"GMT"*, and *"Etc/GMT"* are all non-primary identifiers that resolve to *"UTC"*.</p>
+
+        <p>
+          <dfn id="time-zone-aware">Time zone aware</dfn> ECMAScript implementations support *"UTC"* and local political rules of time zones represented by all other Zone and Link names (and <b>only</b> such names) from the IANA Time Zone Database.
+          An ECMAScript implementation that includes the ECMA-402 Internationalization API <b>must</b> be time zone aware.
+          However, it is recommended that all ECMAScript implementations be time zone aware.
+          Furthermore, even non-time zone aware implementations that do not support the entire IANA Time Zone Database are still recommended to use IANA Time Zone Database identifiers to represent time zones.
+        </p>
+
+        <p>ECMAScript implementations that do not include local political rules for any time zones must support at least one available time zone identifier, *"UTC"*, which must be primary.</p>
+
+        <p>
+          Time zone identifiers in the IANA Time Zone Database can change over time.
```

Comment:
This (particularly the instructions for building tzdata) is perhaps more detail than warranted, at least outside of a Note.

Reply:
Makes sense. I moved this entire paragraph into an `<emu-note>`.